<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlutoSDRInputGUI</class>
 <widget class="QWidget" name="PlutoSDRInputGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>260</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PlutoSDR Input</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_freq">
     <property name="topMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="freqLeftLayout">
       <item>
        <layout class="QHBoxLayout" name="freqLeftTopLayout">
         <item>
          <widget class="ButtonSwitch" name="startStop">
           <property name="toolTip">
            <string>start/stop acquisition</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../sdrgui/resources/res.qrc">
             <normaloff>:/play.png</normaloff>
             <normalon>:/stop.png</normalon>:/play.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ButtonSwitch" name="record">
           <property name="toolTip">
            <string>Toggle record I/Q samples from device</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../sdrgui/resources/res.qrc">
             <normaloff>:/record_off.png</normaloff>
             <normalon>:/record_on.png</normalon>:/record_off.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="freqLeftBotLayout">
         <item>
          <widget class="QLabel" name="adcRateLabel">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>ADC rate before hardware downsampling (k or MS/s)</string>
           </property>
           <property name="text">
            <string>00000k</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="freqLeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ValueDial" name="centerFrequency" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Main center frequency in kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="freqRightLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="freqRightTopLayout">
         <item>
          <widget class="QLabel" name="freqUnits">
           <property name="text">
            <string> kHz</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="freqRightBotLayout">
         <item>
          <widget class="QLabel" name="deviceRateLabel">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Baseband I/Q sample rate kS/s</string>
           </property>
           <property name="text">
            <string>00000k</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="loPPMLabel">
       <property name="text">
        <string>LO ppm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="loPPM">
       <property name="toolTip">
        <string>Local oscillator correction (ppm)</string>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="loPPMText">
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Local oscillator correction (ppm)</string>
       </property>
       <property name="text">
        <string>-00.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="dcOffset">
       <property name="toolTip">
        <string>Automatic DC offset removal</string>
       </property>
       <property name="text">
        <string>DC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="iqImbalance">
       <property name="toolTip">
        <string>Automatic IQ imbalance correction</string>
       </property>
       <property name="text">
        <string>IQ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TransverterButton" name="transverter">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Transverter frequency translation dialog</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_lna">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="decimationLayout">
     <property name="topMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="swDecimLabel">
       <property name="text">
        <string>Sw</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="swDecim">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Software decimation factor</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fcPosLabel">
       <property name="text">
        <string>Fc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fcPos">
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Relative position of device center frequency</string>
       </property>
       <item>
        <property name="text">
         <string>Inf</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sup</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cen</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="antennaLabel">
       <property name="pixmap">
        <pixmap resource="../../../sdrgui/resources/res.qrc">:/antenna.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="antenna">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Input path</string>
       </property>
       <item>
        <property name="text">
         <string>Abal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bbal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cbal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>An</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bn</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cn</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tx1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tx2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tx3</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sampleRateLayout">
     <item>
      <widget class="QLabel" name="samplerateLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>SR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValueDial" name="sampleRate" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Device to host sample rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="samplerateUnit">
       <property name="text">
        <string>S/s</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lpfLabel">
       <property name="text">
        <string>LP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValueDial" name="lpf" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Analog lowpass filer bandwidth (kHz)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lpfUnits">
       <property name="text">
        <string>kHz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_freq">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="firLayout">
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="ButtonSwitch" name="lpFIREnable">
       <property name="toolTip">
        <string>Enable or disable TSP digital FIR lowpass filter</string>
       </property>
       <property name="text">
        <string>FIR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValueDial" name="lpFIR" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Digital FIR lowpass filers bandwidth (kHz) @ ~-6dB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lpfFIRUnits">
       <property name="text">
        <string>kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lpFIRDecimationLabel">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lpFIRDecimation">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>FIR decimation changes lower sample rate limit (see documentation)</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lpFIRGainLabel">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lpFIRGain">
       <property name="toolTip">
        <string>FIR gain (dB)</string>
       </property>
       <item>
        <property name="text">
         <string>-12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>+6</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="gainLayout">
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QComboBox" name="gainMode">
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Gain control mode (Manual, Slow attack AGC, Fast attack AGC, Hybrid AGC) </string>
       </property>
       <item>
        <property name="text">
         <string>Man</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fast</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hyb</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="gain">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Gain setting (dB) in manual mode only</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>77</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gainText">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Gain value (dB)</string>
       </property>
       <property name="text">
        <string>20</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="actualGainText">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Actual gain (dB)</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rssiText">
       <property name="minimumSize">
        <size>
         <width>68</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Device RSSI indication (dB)</string>
       </property>
       <property name="text">
        <string>-100.00 dB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="temperatureText">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Board temperature (degrees C)</string>
       </property>
       <property name="text">
        <string>00C</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="padLayout">
     <item>
      <spacer name="verticalPadSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
  <customwidget>
   <class>TransverterButton</class>
   <extends>QPushButton</extends>
   <header>gui/transverterbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
